controller/run.go

```diff
@@ -31,15 +43,27 @@ func LoadAndRun() error {
 }
 
 // RunUnmanaged runs the shipper out of a local config file without using the control protocol.
-func RunUnmanaged(cfg config.ShipperConfig) error {
+func RunUnmanaged(cfg config.ShipperRootConfig) error {
```

Comment:
Do we have any tests for running in unmanaged mode? What is enforcing that it keeps working?

Reply:
Yah, I've been deprioritizing it a bit, since it took me so long to just get the shipper talking to other components. Will see if I can add a test or two, at least...